cl/simple_pages/templates/help/relative_dates_help.html

```diff
@@ -0,0 +1,75 @@
+
+{% block title %}Help with Relative Dates – CourtListener.com{% endblock %}
+{% block og_title %}Help with Relative Dates – CourtListener.com{% endblock %}
+
+{% block description %}Learn how to use relative dates in CourtListener to make your searches and alerts more dynamic and up-to-date.{% endblock %}
+{% block og_description %}Learn how to use relative dates in CourtListener to make your searches and alerts more dynamic and up-to-date.{% endblock %}
+
+{% block sidebar %}{% endblock %}
+
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="col-xs-12 hidden-md hidden-lg">
+  <h4 class="v-offset-below-2">
+    <i class="fa fa-arrow-circle-o-left gray"></i>
+    <a href="{% url "help_home" %}">Back to Help</a>
+  </h4>
+</div>
+
+<div id="toc-container" class="hidden-xs hidden-sm col-md-3">
+  <div id="toc">
+    <h4 class="v-offset-below-3">
+      <i class="fa fa-arrow-circle-o-left gray"></i>
+      <a href="{% url "help_home" %}">Back to Help</a>
+    </h4>
+    <h3>Table of Contents</h3>
+    <ul>
+      <li><a href="#relative-dates">Relative Dates</a></li>
+      <li><a href="#syntax">Allowed Syntax</a></li>
+      <li><a href="#notes">Technical Notes</a></li>
+    </ul>
+  </div>
+</div>
+
+<div class="col-xs-12 col-md-8 col-lg-6">
+  <h1 id="relative-dates">Help with Relative Dates</h1>
+  <p class="lead">Use relative dates in your queries to keep your searches and alerts dynamically up to date.</p>
+  <p>When placing a search query into CourtListener, you can use either “Calendar” date queries or “Relative” date queries:</p>
+  <ul>
+    <li>Calendar date queries let you filter before or after a given date.</li>
+    <li>Relative dates filter results dynamically based on the date the query is made by a user or system. Instead of specifying a calendar date, such a “January 1, 2025,” you can use relative time periods like <code>1 day ago</code>, <code>Past 1 month</code>, or <code>-14d</code>.</li>
+  </ul>
+
+  <p>Relative date queries are useful for saved or bookmarked queries, so that your query always filters to recent content.
+    They are also valuable when creating alerts. Imagine you want to receive an alert whenever a recent document — filed in the past week — is made available in the  <a href="{% url "advanced_r" %}">RECAP Archive</a>.
+    With relative dates, your date range is rolling, and the “Past Week” changes in sync with the date the alert is triggered.
+  </p>
+
+  <h2 id="syntax">Allowed Syntax</h2>
+  <p>You can use the following syntax:</p>
+  <ul>
+    <li><code>Xd ago</code></li>
+    <li><code>X days ago</code></li>
+    <li><code>-Xd</code></li>
+    <li><code>-Xd ago</code></li>
+    <li><code>-X days</code></li>
+    <li><code>Past X days</code></li>
+  </ul>
+
+  <p>To filter to months or years, substitute <code>m</code>, <code>months</code>, <code>y</code>, or <code>years</code> instead of <code>d</code> or <code>days</code>.</p>
```

Comment:
Assuming this is true, let's add a sentence like:

```suggestion
  <p>To filter to months or years, substitute <code>m</code>, <code>months</code>, <code>y</code>, or <code>years</code> instead of <code>d</code> or <code>days</code>.</p>
  <p>When using the API, relative dates are accepted on all date fields, even if they are not supported by the website.</p>
```

Reply:
Sure, I added the sentence. Just to clarify, this syntax is allowed in any of the fields available in the `SearchForm` class, which are:

RECAP:
`filed_after, filed_before`

Opinions:
`filed_after, filed_before`

OA:
`argued_before, argued_after`

Judges:
`born_before, born_after`


Even though it's only available for RECAP in the frontend.

Reply:
But if that's the case, are we sure that relative dates are accepted on all date fields available in the APIs? For example RECAP Documents can be filtered by `date_created`, but that field is not in the `SearchForm` class.

Reply:
Thanks! You're right, that can be confusing. I just applied a tweak so the sentence now specifically refers to the Search API.

`When using the Search API, relative dates are accepted on all date filters, even if they are not supported by the website.`

Reply:
Awesome, thanks, that's much clearer to me now :) I made a [tiny suggestion](https://github.com/freelawproject/courtlistener/pull/5608#discussion_r2107900338), what do you think?